Automattic/wp-calypso
# Implement monthly to annual upsell page

Fixes https://github.com/Automattic/dotcom-forge/issues/2197

## Proposed Changes

* Implement the page
![upsell](https://user-images.githubusercontent.com/6586048/234832404-4d7cf764-ab99-4c1d-beb3-eb1c308c8f9b.png)

![mobile](https://user-images.githubusercontent.com/6586048/235107678-9bd9d7db-be7a-4664-8b39-45357128388a.png)

## Testing Instructions



* An account on a free plan
* Go to `/plans/monthly/:siteSlug?flags=upsell/monthly-to-annual` to buy a monthly plan (personal/premium/business)
* Should redirect to upsell page on checkout
* Check if the savings price and percentage is correct (we're using monthly * 12 compared to the discounted price at checkout for the annual plan)
* See if anything is weird (misspelled, miscalculation, etc.)
* Test Add to cart and buy it -> thank you page (except for business plan, it has a [double upsell issue](https://github.com/Automattic/dotcom-forge/issues/2230))
* Test Consider later -> thank you page
* Test for all three plans (personal/premium/business)

## Pre-merge Checklist
